frontend/src/v5/ui/routes/viewer/handleTicketsCardSearchParams/handleTicketsCardSearchParams.component.tsx

```diff
@@ -15,18 +15,26 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
The URL params are consumed as the user reaches the viewer to dictate the filters, but then they are no longer used. Should they be cleared from the URL?